cfgov/jinja2/v1/events/_macros.html

```diff
@@ -202,12 +202,16 @@ <h2 class="event-venue_heading">
             <tr>
                 <td>
                     {% import 'macros/time.html' as time %}
-                    {{ time.render(
-                        bound.start_time.render(),
-                        {'date':false,'time':true,'timezone':false}
-                    ) }}
-                    &ndash;
-                    {{ time.render(bound.end_time.render(), {'time':true, 'timezone':true}) }}
+                    {% if bound.start_time | string != 'None' %}
```

Comment:
Change made in https://github.com/cfpb/cfgov-refresh/pull/2739/commits/a4bcd7547a35702ea5197fa49fcafaa89b539a4a